components/SourcesModal.vue

```diff
@@ -0,0 +1,182 @@
+<template>
```

Comment:
This is the other net-new area. This is mostly styling and template code, but contains a bit of scripting to deal with the sources.

Reply:
I was back and forth on that one.... probably should. Maybe I'll stub something up with a slot content to clean this up.